src/logging_vnode.erl

```diff
@@ -714,7 +714,7 @@ handle_commit(TxId, OpPayload, T, Key, MinSnapshotTime, Ops, CommittedOpsDict) -
 	    NewCommittedOpsDict = 
 		lists:foldl(fun(#update_log_payload{key = KeyInternal, type = Type, op = Op}, Acc) ->
 				    case ((MinSnapshotTime == undefined) orelse
-									   (not vectorclock:gt(SnapshotTime, MinSnapshotTime))) of
+									   (not vectorclock:all_dots_greater(SnapshotTime, MinSnapshotTime))) of
```

Comment:
This one is a bit tricky, this function is used to return the list of ops needed to generate a snapshot from the log.

It doesn't actually generate the snapshot here, but sends the list of ops to the materializer, which then generates the snapshot.

It looks like the comparison used here will return more ops than necessary, but this is not a problem since the extra ops will be trimmed by the materializer.  (I'm not sure if this was the original intention or not though).

This is similar to the discussions with Santiago on the relation between the materialzer and log for generating snapshots.  So probably what we should do it just be consistent on the leveldb and the disk_log so the functions give the same thing to the materializer.